src/executables/utility.h

```diff
@@ -1241,6 +1241,8 @@ typedef struct _ha_config
 #define PLANDUMP_DROP_L                         "drop"
 #define PLANDUMP_OUTPUT_FILE_S		        'o'
 #define PLANDUMP_OUTPUT_FILE_L                  "output-file"
+#define PLANDUMP_SHA1_S		        	's'
+#define PLANDUMP_SHA1_L                  	"sha1"
```

Comment:
The '-s' option can also be used to view the plan for a given SHA1.
How about changing the option name?

Reply:
Good point. But I don't want to do it because I have to edit the entire message.